.github/workflows/release.yml

```diff
@@ -26,27 +26,27 @@ jobs:
     steps:
       - uses: actions/checkout@v4.1.1
       - name: Download artifacts for ankaios-linux-amd64-bin
-        uses: actions/download-artifact@v3.0.2
+        uses: actions/download-artifact@v4.1.6
```

Comment:
The latest version of the download artifact is 4.1.7 now. Is there any reason why to use 4.1.6?